Components/Queues/AsyncQueueConsumer.php

```diff
@@ -4,90 +4,170 @@
 
 namespace Smartbox\Integration\FrameworkBundle\Components\Queues;
 
-use Psr\Log\LoggerAwareInterface;
-use Psr\Log\LoggerAwareTrait;
-use Smartbox\Integration\FrameworkBundle\Components\Queues\Handler\AmqpQueueHandler;
-use Smartbox\Integration\FrameworkBundle\Core\Consumers\ConsumerInterface;
-use Smartbox\Integration\FrameworkBundle\Core\Consumers\IsStopableConsumer;
+use PhpAmqpLib\Message\AMQPMessage;
+use Smartbox\CoreBundle\Type\SerializableArray;
```

Comment:
`SerializableArray` is not used in the class, so it should be removed

Reply:
Removed